homeassistant/components/velux/translations/en.json

```diff
@@ -0,0 +1,16 @@
+{
+    "config": {
+        "error": {
+            "invalid_auth": "Authenticatie ongeldig"
```

Comment:
It seems you mixed up the `en.json` and the `nl.json`. Please remove your translations files, since it will be added via Lokalize.

You can keep en.json to make testing easier, but I would advise to generate it via `python3 -m script.translations develop` based on your strings.json.